src/dispatchError.ts

```diff
@@ -6,7 +6,11 @@ export function extractDispatchErrorDescription(dispatchError: DispatchError): s
       const module = dispatchError.asModule;
       const error = dispatchError.registry.findMetaError(module);
 
-      return `${error.section}.${error.name}: ${error.docs[0]}` ?? `${error.section}.${error.name}`;
```

Comment:
👍 
Wow, this line did not make any sense. The string `${error.section}.${error.name}: ${error.docs[0]}` can never be `null` or `undefined` so that the nullish coalescing operator did not have an effect.